packages/dappmanager/src/daemons/telegramBot/buildTelegramMessage.ts

```diff
@@ -0,0 +1,56 @@
+import { TelegramMessageHeader } from "./types";
+
+/**
+ * Returns the message formatted with the header
+ * @param param0
+ */
+export function buildTelegramMessage({
+  header,
+  telegramMessage
+}: {
+  header: TelegramMessageHeader;
+  telegramMessage: string;
+}): string {
+  const head = formatTelegramHeader(header);
+  return bold("DAppNode ") + head + telegramMessage;
+}
+
+/**
+ * Builds the telegram header
+ * @param param0
+ */
+function formatTelegramHeader(header: TelegramMessageHeader): string {
+  switch (header) {
+    case "Alert":
+      return `❌ 📢 ${bold(header)}❗:\n`;
+    case "Danger":
+      return `⚠ ${bold(header)} ⚡:\n`;
```

Comment:
Use same emoji left and right in `Alert` and `Danger` too for consistency and looks cleaner